common/source/docs/common-dock-mode.rst

```diff
@@ -0,0 +1,103 @@
+.. _common-dock-mode:
+
+=========
+Dock Mode
+=========
+
+The DOCK mode maneuvers a vehicle towards a stationary docking target automatically.
+Currently, we support docking with the help of a camera attached to a companion computer.
+The companion computer estimates the pose of the marker (ArucoMarker/AprilTag) placed at the docking station and sends
+`LANDING_TARGET <https://mavlink.io/en/messages/common.html#LANDING_TARGET>`__ mavlink messages to the vehicle which are then processed by the precision landing library
+(`AC_PrecLand <https://github.com/ArduPilot/ardupilot/tree/master/libraries/AC_PrecLand>`__) to estimate the position of the docking target.
+
+Successful tests have been conducted using the OpenMV camera as a companion computer to perform auto-docking.
+However, any companion computer equipped with an RGB camera (like OpenMV camera, Raspberry Pi, Jetson Nano etc.) which can estimate pose
+of marker placed at docking target and send mavlink messages to the vehicle can be used.
+
+The OpenMV Camera
+=================
+
+OpenMV camera is a pocket sized camera board which is capable of performing various computer vision tasks.
+It can transmit mavlink messages over UART interface and hence it can be used as a companion computer to perform auto-docking.
```

Comment:
```suggestion
It can transmit the required mavlink messages over its UART interface and hence it can be used as a companion computer to perform auto-docking.
```